scheduler/test/cook/test/simulator.clj

```diff
@@ -574,7 +593,7 @@
   (not (seq (trace-diffs trace-a trace-b))))
 
 (deftest test-simulator
-  (let [users ["a" "b" "c" "d"]
+  (time (let [users ["a" "b" "c" "d"]
```

Comment:
Nitpick: this causes formatting to be off for the rest of the form.

Reply:
Removed the time